client/src/components/LanguageOptions.js

```diff
@@ -1,25 +1,13 @@
 const languageOptions = [
-  { key: "Ar", text: "Arabic", value: "Arabic" },
-  { key: "Chi", text: "Chinese", value: "Chinese" },
-  { key: "Dan", text: "Danish", value: "Danish" },
-  { key: "Dut", text: "Dutch", value: "Dutch" },
-  { key: "En", text: "English", value: "English" },
-  { key: "Fre", text: "French", value: "French" },
-  { key: "Ger", text: "German", value: "German" },
-  { key: "Grek", text: "Greek", value: "Greek" },
-  { key: "Hun", text: "Hungarian", value: "Hungarian" },
-  { key: "Ita", text: "Italian", value: "Italian" },
-  { key: "Jap", text: "Japanese", value: "Japanese" },
-  { key: "Kor", text: "Korean", value: "Korean" },
-  { key: "Lith", text: "Lithuanian", value: "Lithuanian" },
-  { key: "Per", text: "Persian", value: "Persian" },
-  { key: "Pol", text: "Polish", value: "Polish" },
-  { key: "Por", text: "Portuguese", value: "Portuguese" },
-  { key: "Rus", text: "Russian", value: "Russian" },
-  { key: "Spa", text: "Spanish", value: "Spanish" },
-  { key: "Swe", text: "Swedish", value: "Swedish" },
-  { key: "Tur", text: "Turkish", value: "Turkish" },
-  { key: "Viet", text: "Vietnamese", value: "Vietnamese" }
+  { key: "zh-cn", text: " MandarinChinese", value: "Mandarin Chinese" },
+  { key: "ar", text: "Arabic", value: "Arabic" },
+  { key: "es", text: "Spanish", value: "Spanish" },
+  { key: "ml", text: "Danish", value: "Malaysian" },
+
+  { key: "en", text: "English", value: "English" },
+  { key: "ru", text: "Russian", value: "Russian" },
+  { key: "be", text: "Russian", value: "Bengali" },
+  { key: "Fr", text: "French", value: "French" }
```

Comment:
change the key Fr to lower case: fr
Every language code should be in lower case

Reply:
I changed it, in the database only have code and name, but in the options we have key, text and value, should i change it?
when i was testing it on the frondend is working but